my-app/src/components/Nav/Nav.css

```diff
@@ -0,0 +1,40 @@
+:root {
+  --white-color: white;
+  --header-style-color:black;
+}
+.header {
+  grid-area: head;
+  padding: 0px 0 0 28px;
```

Comment:
use rems when value is more than 10

Reply:
Okay. will change it during refactoring.